application/src/main/java/org/togetherjava/tjbot/config/Config.java

```diff
@@ -123,6 +125,7 @@ private Config(@JsonProperty(value = "token", required = true) String token,
         this.helperPruneConfig = Objects.requireNonNull(helperPruneConfig);
         this.featureBlacklistConfig = Objects.requireNonNull(featureBlacklistConfig);
         this.selectRolesChannelPattern = Objects.requireNonNull(selectRolesChannelPattern);
+        this.cakeDayConfig = cakeDayConfig;
```

Comment:
Missing `Objects.requireNonNull`

Reply:
Added it! ✅

Reply:
> Added it! ✅

Instead of always writing "Added"/"Done" or smth just simply press the `Resolve conversation` button <3

Reply:
> > Added it! ✅
> 
> Instead of always writing "Added"/"Done" or smth just simply press the `Resolve conversation` button <3

Okay! 🚀

Jokes aside, I always let the OP resolve the conversations so that they don't unresolve it later to add more stuff

Reply:
> Jokes aside, I always let the OP resolve the conversations so that they don't unresolve it later to add more stuff. I don't know if that's the right way to do it though.

I prefer if you just do it